pkg/platform/model/vulnerabilities.go

```diff
@@ -1,16 +1,35 @@
 package model
 
 import (
+	"fmt"
+	"sort"
+
 	"github.com/ActiveState/cli/internal/errs"
 	"github.com/ActiveState/cli/pkg/platform/api/vulnerabilities"
 	"github.com/ActiveState/cli/pkg/platform/api/vulnerabilities/model"
 	"github.com/ActiveState/cli/pkg/platform/api/vulnerabilities/request"
 	"github.com/ActiveState/cli/pkg/platform/authentication"
 )
 
-type VulnerabilityIngredient request.Ingredient
+type VulnerabilityIngredient struct {
+	Name             string
+	PrimaryNamespace string
+	Version          string
+	Vulnerabilities  *Vulnerabilites
+}
+
+type Vulnerabilites struct {
+	Critical []string
+	High     []string
+	Medium   []string
+	Low      []string
+}
 
-func FetchVulnerabilitiesForIngredients(auth *authentication.Auth, ingredients []*VulnerabilityIngredient) ([]model.VulnerableIngredientsFilter, error) {
+func (v Vulnerabilites) Length() int {
+	return len(v.Critical) + len(v.High) + len(v.Medium) + len(v.Low)
+}
+
+func FetchVulnerabilitiesForIngredients(auth *authentication.Auth, ingredients []*request.Ingredient) ([]VulnerabilityIngredient, error) {
```

Comment:
Nit: Fetch to me says it's going to make an API call.

Reply:
It is, just not in the changeset: https://github.com/ActiveState/cli/pull/3051/files#diff-1dec3c13927fe0d61f8457d8c7951d7a60d655d643f1aeff2a37a59bd0b41031R46